pkg/controller/common/association/resources.go

```diff
@@ -0,0 +1,74 @@
+package association
```

Comment:
This whole file is much easier to digest since you changed the labelling logic 🙏